blackjack_game/index.js

```diff
@@ -0,0 +1,65 @@
+let player = {
+    name: "Per",
+    chips: 200
+}
+
+let cards = []
+let sum = 0
+let hasBlackJack = false
+let isAlive = false
+let message = ""
+let messageEl = document.getElementById("message-el")
+let sumEl = document.getElementById("sum-el")
+let cardsEl = document.getElementById("cards-el")
+let playerEl = document.getElementById("player-el")
+
+playerEl.textContent = player.name + ": $" + player.chips
+
+function getRandomCard() {
+    let randomNumber = Math.floor( Math.random()*13 ) + 1
```

Comment:
Lint error. You should have spaces around "*". And no space before Math.random().